### PR TITLE
Ietf energy carlos

### PR DESCRIPTION
**Pull request by cpignata**
Complete set of content additions and editorial fixes.

### CODE REVIEW

File: draft-eckert-ietf-and-energy-overview.mkd [hunk 1]
```diff
@@ -217,6 +264,8 @@ interested in it, in the hope to raise more interest in energy-related
 activities within the IETF, such as identifying gaps and investigating
 solutions as appropriate.
 
+This document captures work until 12/2022, at which time the "IAB workshop on Environmental Impact of Internet Applications and Systems" (https://www.iab.org/activities/workshops/e-impact/) created new interest and work in the topic.
```
Comment (boucadair) [inline draft-eckert-ietf-and-energy-overview.mkd]:
```suggestion
This document captures work until 12/2022, at which time the "IAB workshop on Environmental Impact of Internet Applications and Systems" revived interest and triggered new work in the topic within the IETF/IRTF. 
```

File: draft-eckert-ietf-and-energy-overview.mkd [hunk 2]
```diff
@@ -234,10 +283,14 @@ for example, by being most narrow.
 
 This memo usually refers to the technologies by significant early RFC or specific draft version,
 as opposed to the newest. This is contrary to the common practice in IETF documents to
-refer to the newest version. This appraoch is meant to allow readers to better understand
+refer to the newest version. This approach is meant to allow readers to better understand
 the historic timeline in which a specific technology was proposed or introduced. Especially successful
 IETF technologies will have newer RFC that updates such initial work.
 
+
+This document captures work until 12/2022, at which time the "IAB workshop on Environmental Impact of Internet Applications and Systems" (https://www.iab.org/activities/workshops/e-impact/) created new interest and work in the topic.
```
Comment (boucadair) [inline draft-eckert-ietf-and-energy-overview.mkd]:
```suggestion
This document captures work until 12/2022, at which time the "IAB workshop on Environmental Impact of Internet Applications and Systems" {{?I-D.iab-ws-environmental-impacts-report}} revived interest and highlighted the need for some structured activity within the IETF/IRTF. 
```

File: draft-eckert-ietf-and-energy-overview.mkd [hunk 3]
```diff
@@ -280,28 +333,28 @@ introduced in the past decades with the Internet and the hyper-scaling that the
...
+most widely deployed packet networking protocols, they were the ones who at least during the 1990s started
 to break away from other protocols both in scale of deployment, as well as in development of further technologies
 to support this scaling.
 
 ### End-to-End Transport
 
 At the core of scalability, even up to now, is the lightweight per-packet-processing enabled through
 end-to-end congestion loss management architecture as embodied through the IETF "Transmission Control Protocol"
-(TCP, {{RFC793}} and successors, e.g. {{I-D.ietf-tcpm-rfc793bis}}). This model eliminated more expensive per-hop, per-packet processing, such as
+(TCP, {{RFC793}} and successors, e.g., {{I-D.ietf-tcpm-rfc793bis}}). This model eliminated more expensive per-hop, per-packet processing, such as
```
Comment (boucadair) [inline draft-eckert-ietf-and-energy-overview.mkd]:
```suggestion
(TCP, {{?RFC9293}}). This model eliminated more expensive per-hop, per-packet processing, such as
```

File: draft-eckert-ietf-and-energy-overview.mkd [hunk 4]
```diff
@@ -513,23 +566,38 @@ into account and using CO2 emission equivalents as its core metric. That paper c
...
 In summary: Telecollaboration has a higher sustainability benefit compared to travel than just
 the comparison of energy consumption because of the higher challenge to use renewable energy
 in transportation than in networking, and this is most extreme in the case of telecollaboration
-that replaces air travel because of the even higher global warming impact of using fossile fuels
+that replaces air travel because of the even higher global warming impact of using fossil fuels
 in air travel.
 
 # Energy Optimization in Specific Networks
 
+## Analysis of routing protocol (in)efficiencies
```
Comment (boucadair) [inline draft-eckert-ietf-and-energy-overview.mkd]:
```suggestion
## Analysis of Routing Protocol (In)Efficiencies
```

File: draft-eckert-ietf-and-energy-overview.mkd [hunk 5]
```diff
@@ -548,7 +616,7 @@ large packets at low bitrates, packet bundling to only consume radio energy at s
 periods, radio energy tuning to just reach the destination(s), minimization of multicasting
 to eliminate need of radio receivers to consume energy and so on. {{RFC8352}} gives a more detailed
 overview, especially because different L2 technologies such as IEEE 802.15.4 type (low power)
-wireless networks, Bluetooth Low Energy (BLE), WiFi (IEEE 802.11) and DEC ULE. 
+wireless networks, Bluetooth Low Energy (BLE), Wi-Fi (IEEE 802.11) and DEC ULE.
```
Comment (boucadair) [inline draft-eckert-ietf-and-energy-overview.mkd]:
```suggestion
wireless networks, Bluetooth Low Energy (BLE), WLAN (IEEE 802.11) and DEC ULE. 
```

File: draft-eckert-ietf-and-energy-overview.mkd [hunk 6]
```diff
@@ -558,22 +626,30 @@ The "IPv6 over Low power WPAN (Wireless Personal Area Networks)" (6lowpan) WG ra
...
+Outside the 6LOWPAN WG, {{RFC9139}} connects Information-Centric Networking (ICN) to Low-Power Wireless Personal Area Networks (LoWPANs).
+
+
 ### LPWAN WG
 
-Since 2014, the "IPv6 over Low Power Wide-Area Networks" (LPWAN) WG has produced 4 RFC
+Since 2014 and before 2023, the "IPv6 over Low Power Wide-Area Networks" (LPWAN) WG has produced 4 RFC
 for low-power wide area networks, such as LoRaWAN {{https://en.wikipedia.org/wiki/LoRa}},
-with three standards, {{RFC8724}}, {{RFC8824}}, {{RFC9011}}.
+with three Standards-Track RFC documents, {{RFC8724}}, {{RFC8824}}, {{RFC9011}}.
```
Comment (boucadair) [inline draft-eckert-ietf-and-energy-overview.mkd]:
```suggestion
with three Standards-Track RFC documents, {{RFC8724}}, {{RFC8824}}, and {{RFC9011}}.
```

File: draft-eckert-ietf-and-energy-overview.mkd [hunk 7]
```diff
@@ -584,29 +660,40 @@ specifications, information models, cross-adaptation layer specification (such a
...
 In the RouTinG (RTG) area of the IETF, the "Routing Over Low power and Lossy networks" (ROLL) WG
 has produced since 2008 23 RFC. Initially it produced requirement RFCs of different type of
 "Low-power and Lossy Networks": urban: {{RFC5548}}, industrial {{RFC5673}}, home automation
 {{RFC5826}} and building automation {{RFC5867}}.
 
- Since then its work is mostly focused
-on the "IPv6 Routing Protocol for Low-Power and Lossy Networks" (RPL) {{RFC6550}}, which
-is used in a wide variety of the above described IPv6 instances of LLN networks
+Since then, its work is mostly focused
+on the "IPv6 Routing Protocol for Low-Power and Lossy Networks" (RPL) {{RFC6550}} {{RFC6551}} {{RFC6552}}, which
```
Comment (boucadair) [inline draft-eckert-ietf-and-energy-overview.mkd:678]:
```suggestion
on the "IPv6 Routing Protocol for Low-Power and Lossy Networks" (RPL) {{RFC6550}} {{RFC6551}}{{RFC6552}}, which
```

**Review by boucadair — CHANGES_REQUESTED** [review]
Thanks Carlos.

File: draft-eckert-ietf-and-energy-overview.mkd (hunk 7 above)
Comment (cpignata) [reply draft-eckert-ietf-and-energy-overview.mkd:678]: I'd leave the extra space between references.

**Comment by cpignata** [comment]
Thanks, Med @boucadair , for the review and requested changes! All incorporated on this PR.

**Review by cpignata — COMMENTED** [review]
Hi, @boucadair Med, these are simply the make targets for xml and txt -- no changes. 
I incorporated all your review comments -- thanks! 
Please let me know if anything else might be needed.
@cpignata Carlos.

**Review by boucadair — APPROVED** [review]

**Pull request merged**